.github/workflows/stale.yml

```
@@ -0,0 +1,25 @@
+name: 'Close stale issues'
+on:
+  schedule:
+    # 1:30am every day
+    - cron: '30 1 * * *'
+
+permissions:
+  issues: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@3de2653986ebd134983c79fe2be5d45cc3d9f4e1
+        with:
+          days-before-stale: 90
+          days-before-close: 30
+          stale-issue-label: stale
+          operations-per-run: 1750
+          stale-issue-message: >
+            This issue has been automatically marked as stale. **If this issue is still affecting you, please leave any comment** (for example, "bump"), and we'll keep it open. If you have any new additional information—in particular, if this is still reproducible in the [latest version of Electron](https://www.electronjs.org/releases/stable) or in the [beta](https://www.electronjs.org/releases/beta)—please include it with your comment!
+          close-issue-message: >
+            This issue has been closed due to inactivity, and will not be monitored.  If this is a bug and you can reproduce this issue on a [supported version of Electron](https://www.electronjs.org/docs/latest/tutorial/electron-timelines#timeline) please open a new issue and include instructions for reproducing the issue.
+          exempt-issue-labels: discussion,security 🔒,enhancement :sparkles:
```

ugh this doesn't parse. you were totes right about unicode in yaml. wtf, this is 2022.
```suggestion
          exempt-issue-labels: "discussion,security \ud83d\udd12,enhancement :sparkles:"
```